app/views/child_benefit_tax/main.html.erb

```diff
@@ -73,10 +73,17 @@
 
       <div class="question-wrapper">
         <% question_four = capture do %>
+          <p class="govuk-list">
```

Comment:
Did you mean to put this class here? I think [it should only be used on `<ul>` elements](https://design-system.service.gov.uk/styles/typography/#lists), not paragraphs. Same comment on line 82.

Reply:
You're right, [it should be `govuk-body`](https://design-system.service.gov.uk/styles/typography/#paragraphs).